### PR TITLE
Inserting blog to db

### PR DESCRIPTION
**Pull request by AdkaB**

### CODE REVIEW

File: src/models/blog.js [hunk 1]
```diff
@@ -10,37 +10,35 @@ client.connect(err => {
 
 //mongodb+srv://User1:<password>@test.fnhqc.mongodb.net/myFirstDatabase?retryWrites=true&w=majority
 
+// const mongoose = require('mongoose')
```
Comment (mhalajova) [inline src/models/blog.js]: remove commented line

File: src/models/blog.js [hunk 2]
```diff
@@ -10,37 +10,35 @@ client.connect(err => {
...
+    data: Buffer,
+    contentType: String,
+  },
+});
+
+export const blogModel = mongoose.model("blogs", blogSchema);
 
-module.exports = mongoose.model('Blogs', blogSchema);
\ No newline at end of file
+export default blogModel;
```
Comment (mhalajova) [inline src/models/blog.js]: I also wrote it to Arsen. But I would export both schema and model.

```suggestion
export default { blogModel, blogSchema as Blog };
```

File: src/services/blogs.js [hunk 3]
```diff
@@ -1,3 +1,30 @@
-// add code here
+import Blog from "../models/blog";
+import { param, validationResult } from "express-validator";
 
-export default {}
+//inserting new blog to DB
+
+const addBlog = async (req, res) => {
+  try {
+    const validationResults = validationResult(req);
```
Comment (mhalajova) [inline src/services/blogs.js:8]: add new line after this

File: src/services/blogs.js [hunk 4]
```diff
@@ -1,3 +1,30 @@
...
+        author: req.body.author,
+        text: req.body.text,
+      });
+      await blog.save();
+      res.status(200);
+      res.json(blog);
+    } else {
+      req.log.info(`validation error value: ${validationResults}`);
+      res.status(400);
+      res.send("validationError");
```
Comment (mhalajova) [inline src/services/blogs.js:24]: Send `validationResults` as a response as well

File: src/services/blogs.js [hunk 5]
```diff
@@ -1,3 +1,30 @@
...
+    }
+  } catch (error) {
+    res.sendStatus(500);
+    res.send("Error!");
+    console.log(error);
```
Comment (mhalajova) [inline src/services/blogs.js]: Write the error to the logger via `req.log.error` instead of `console.log`

File: src/services/blogs.js [hunk 6]
```diff
@@ -1,3 +1,30 @@
...
+
+const addBlog = async (req, res) => {
+  try {
+    const validationResults = validationResult(req);
+    if (validationResults.isEmpty()) {
+      const blog = new Blog({
+        title: req.body.title,
+        author: req.body.author,
+        text: req.body.text,
+      });
```
Comment (mhalajova) [inline src/services/blogs.js:15]: add new line after this

File: src/services/blogs.js [hunk 7]
```diff
@@ -1,3 +1,30 @@
...
+      await blog.save();
```
Comment (mhalajova) [inline src/services/blogs.js:17]: add new line after this

File: src/models/blog.js [hunk 8]
```diff
@@ -10,37 +10,35 @@ client.connect(err => {
 
 //mongodb+srv://User1:<password>@test.fnhqc.mongodb.net/myFirstDatabase?retryWrites=true&w=majority
```
Comment (jakubarendac) [inline src/models/blog.js]: this shouldn't belong here. If you want to have any information for devs how to connect to DB please put it inside some docs or readme

File: src/routes/blogs.js [hunk 9]
```diff
@@ -1,17 +1,18 @@
...
-    res.send('We are on blogs');
-});
-
-router.post('/', (req, res) => {
-    console.log(req.body);
-});
+router.post(
+  "/",
+  body("title")
+    .isLength({ min: 3, max: 80 })
```
Comment (jakubarendac) [inline src/routes/blogs.js:11]: do we have this validation in schema ? If not add it there, otherways it doesn't make sense

**Review by jakubarendac — CHANGES_REQUESTED** [review]

**Pull request merged**